code/game/machinery/body_scanner.dm

```diff
@@ -523,15 +523,19 @@
 
 		if (O.implants.len)
 			var/unk = 0
+			var/list/organic = list()
 			for (var/atom/movable/I in O.implants)
-				if (is_type_in_list(I, known_implants))
+				if(is_type_in_list(I, known_implants))
 					wounds += "\a [I.name] is installed."
 				if(istype(I, /obj/effect/spider))
-					wounds += "Abnormal organic body detected."
-				else
+					organic += I
+				if(!is_type_in_list(I, known_implants) && !istype(I, /obj/effect/spider))
 					unk += 1
```

Comment:
So what you can do in this case, instead of that last check, which duplicates the first two checks, is do an if, else if, else. If in known implants, add, else if it's a spider, add, else it's neither of those, so it's unknown. This should also ensure there's no "ghost" abnormal mass.

Reply:
I tried to do that originally and the problem became that if a limb had all three (or even just two), then one of the items would override everything else. I couldn't figure out a better way of making sure one wouldn't override the other without intensive shitcoding, so while ugly, the last if works for now

Reply:
Oh, hold up. I'm a fucking idiot. I forgot rejuvenating also removes the implants and that might've fucked with my original testing with those

Reply:
Yeah what you said works. I'm. So sorry. I'll update the lines soon!